resources/crds/ripsaw_v1alpha1_cyclictest_cr.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: cyclictest
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: <ES_SERVER>
+    port: <ES_PORT>
+  workload:
+    name: "cyclictest"
+    args:
+      duration: "1m"
```

Comment:
the default in code is `1h`, but the default in our cr is `1m`?

Reply:
Yeh I'd prefer to keep it in the minutes as a quick smoke test and then run the longer version.

I think we can keep both at 5m, thoughts?

Reply:
Done.